src/main/java/core/basesyntax/PurchaseOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
```

Comment:
Make sure to organize your classes into packages based on their responsibilities. For example, this `PurchaseOperation` class could be placed in a `strategy` package since it's related to the Strategy pattern.